client/src/store/cards/cards.reducer.ts

```diff
@@ -24,6 +24,7 @@ export const cardsReducer = (
     case CardsActionTypes.LoadCardsError:
       return {
         ...state,
+        cards: state.cards,
```

Comment:
unnecessary, spread with state (...state) do the same

Reply:
Fixed